### PR TITLE
CBBP-1061 Use make reqs-install to install application python requirements

### PR DESCRIPTION
**Pull request by rnagle**
Works with the changes included in https://github.com/CMSgov/bluebutton-web-server/pull/589.

### CODE REVIEW

**Comment by rnagle** [comment]
So this started out as a simple PR to update the `pip install ...` step to use the vendored requirements in the web server repo, but during a test build I noticed several other things needed some attention.

Other things included here:
- Removes snmp playbook from the build process since the file it modifies is no longer present in the base AMI we're using
- Pins the system `pip` version to `9.0.3` since v10 was causing errors
- Removes some debugging/convenience ansible tasks at the end of the build that are of minimal utility and really only add to the time required for a build
- Moves umask and yum workaround steps to the `base_patch` role since this seems like the logical grouping for these items
- Stop splunk during the build and let it (re)start itself when the instance comes back up

File: Jenkinsfiles/Jenkinsfile.build_ami [hunk 1]
```diff
@@ -160,8 +141,15 @@ pipeline {
                 file(credentialsId: vault_pass, variable: 'vp')
               ]) {
                 sh """
+                  virtualenv -ppython2.7 venv
```
Comment (samgensburg-gov) [inline Jenkinsfiles/Jenkinsfile.build_ami]: Is there a reason not to use Python 3 to match our application servers?

Comment (samgensburg-gov) [reply Jenkinsfiles/Jenkinsfile.build_ami]: Hmm, I see now that this is copied from above. Still an interesting question though.

**Review by samgensburg-gov — APPROVED** [review]
I'm not sure I completely understand the splunk changes, but this looks good. It's amazing how much this allowed us to remove.

File: Jenkinsfiles/Jenkinsfile.build_ami (hunk 1 above)
Comment (rnagle) [reply Jenkinsfiles/Jenkinsfile.build_ami]: I'll give this a shot, it shouldn't harm anything.

Comment (rnagle) [reply Jenkinsfiles/Jenkinsfile.build_ami]: Tested and it worked fine! 👍

**Pull request merged**